components/front-page/Topics.tsx

```diff
@@ -21,11 +21,14 @@ export const Topics = ({
   return (
     <>
       <div className="hidden w-full gap-3 md:flex">
-        {topics.map((topic, index) => (
-          <Link
-            key={index}
-            href={`/topics/${topic.slug}`}
-            className={`
+        {topics.map((topic, index) => {
+          return (
+            //TODO: temp solution. waiting for login
```

Comment:
What do you mean by `waiting for login`? Do you need login access to the content management system, in which you can deactivate topics on the front end?